opts/charts.go

```diff
@@ -324,6 +324,31 @@ type LineChart struct {
 
 	// Whether to show symbol. It would be shown during tooltip hover.
 	ShowSymbol bool
+
+	// Icon types provided by ECharts includes
+	//  'circle', 'rect', 'roundRect', 'triangle', 'diamond', 'pin', 'arrow', 'none'
+	// Full documentation: https://echarts.apache.org/en/option.html#series-line.symbol
+	Symbol string
+
+	// symbol size. It can be set to single numbers like 10, or use an array to represent width and height. For example, [20, 10] means symbol width is 20, and height is10.
+	// Full documentation: https://echarts.apache.org/en/option.html#series-line.symbolSize
+	SymbolSize float32
```

Comment:
If there is `float`, it seems we can not set a array as the comments mentioned.

Reply:
Yes, there are some limitations because of Go static types. 
This is relevant for multiple configuration options, which can be different types in original Apache Echarts. 
What I've seen in current `go-echarts` implementation, Go wrapper mostly covers one of the options. (E.g )
I think float here for size is good enough for most of the cases.

Reply:
I've pushed fix like you suggested